helm/cluster-cloud-director/templates/_ssh.tpl

```diff
@@ -0,0 +1,40 @@
+# The helper functions here can be called in templates and _helpers.tpl
+# This file should be self-sufficient. Don't call any functions from _helpers.tpl
+
+
+{{- define "sshFiles" -}}
+{{- if $.Values.ssh.trustedCaKeys -}}
+- path: /etc/ssh/trusted-user-ca-keys.pem
+  permissions: "0600"
+  content: |
+    {{- range $.Values.ssh.trustedCaKeys}}
+    {{.}}
+    {{- end }}
+- path: /etc/ssh/sshd_config
+  permissions: "0600"
+  content: |
+    {{- .Files.Get "files/etc/ssh/sshd_config" | nindent 4 }}
+{{- end -}}
+{{- end }}
+
+{{- define "sshPostKubeadmCommands" -}}
+- systemctl restart sshd
+{{- end -}}
+
+{{- define "sshUsers" -}}
+{{- if $.Values.ssh.users -}}
+users:
+{{- range $.Values.ssh.users }}
+- name: {{ .name }}
+{{- if .sudo }}
+  sudo: {{ .sudo }}
+{{- end }}
+{{- if .authorizedKeys }}
+  sshAuthorizedKeys:
+  {{- range .authorizedKeys }}
+  - {{ . }}
+  {{- end }}
+{{- end }}
+{{- end -}}
+{{- end }}
+{{- end -}}
```

Comment:
I feel like this https://github.com/giantswarm/cluster-cloud-director/pull/81#discussion_r1088313086 is still an issue. I won't block on that though, but I still think that would simplify the chart a lot.

Reply:
I updated in this way.